specs/serving/knative-api-specification-1.0.md

```diff
@@ -1302,9 +1302,9 @@ Max: 100
    </td>
    <td>The <code>percent</code> is optionally used to specify the percentage of requests which should be allocated from the main Route domain name to the specified <code>revisionName</code> or <code>configurationName</code>.
 <p>
-To indicate that percentage based routing is to be used, at least one <code>traffic</code> section MUST have a non-zero <code>percent</code> value, and all values MUST sum to 100. Note, a missing <code>precent</code> value implies zero.
+To indicate that percentage based routing is to be used, at least one <code>traffic</code> section MUST have a non-zero <code>percent</code> value, and all values MUST sum to 100. Note, a missing <code>percent</code> value implies zero.
    </td>
-   <td>OPTIONAL
+   <td>REQUIRED
```

Comment:
Doesn't make this required mean that someone can't support other traffic splitting mechanisms?
If I'm following, it seems like perhaps you want a MUST some place else to indicate that %-based routing MUST be supported - not that the syntax MUST have a % value. No?

Reply:
> Doesn't make this required mean that someone can't support other traffic splitting mechanisms?

If we make this OPTIONAL then you can have conformant distributions of Knative that don't support traffic splitting. I don't think we want that.

Thus we should look at the spec from a different perspective 

ie. rather than reading this as
> This field is REQUIRED when I create the object

it should be read as
>  This platform is REQUIRED to accept this field

Reply:
Ah ok - I misread it as it was a "REQUIRED to be present" field. I didn't expand things enough to see that it's a `Schema Requirement`. Sorry about missing that.